src/util/psbt/map/input.rs

```diff
@@ -183,8 +183,8 @@ impl PsbtSigHashType {
         }
     }
 
-    /// Obtains the inner sighash byte from this [`PsbtSigHashType`].
-    pub fn inner(self) -> u32 {
+    /// Obtains the `u32` sighash byte from this [`PsbtSigHashType`].
+    pub fn as_u32(self) -> u32 {
```

Comment:
Did you mean to write `to_u32`? You wrote it in the PR comment that way and I think it should be `to_u32` because it doesn't return a reference and performs copy.

While we're renaming things shouldn't this be something like `to_consensus_u32`? (I'm not sure which representation it is.) And maybe fix the doc claiming it returns "byte" while you're touching that part.

Reply:
I went with `to_u32` because that is Rust convention. This PR highlights that we have a bunch of these as/to/into things wrong. I'll do the others as a separate PR.

Reply:
Follow up PR #798

Reply:
Why not name the representation? We were using names that clearly state the representation elsewhere and I think it's very useful. I wouldn't even say it breaks Rust convention. (If it does then this would be the exceptional case when I disagree with the convention.)

Reply:
My bad for not responding, I did have a reason. A `EcdsaSigHashType` could have been constructed with either `from_u32_consensus` _or_ `from_u32_standard`. When one calls `to_u32` on that object we do not know which it is standard or consensus so I figured `to_u32` was the best name. Thoughts?

Reply:
Well, it's consensus *encoding* (we did not map it to our internal representation). `from_u32_standard` is also consensus encoding, it's just more restrictive.

Because our type does funny things to non-standard values, I believe it should be called `to_u32_standard`. Also standard implies consensus.

Reply:
I'm hoping this PR can go into the 0.28 release. Can we leave the improvement of `to_u32` -> `to_u32_standard` until next release so that we can change `EcdsaSigHashType::as_u32` with it?

Reply:
I've added a link to this thread to my todo list so it doesn't get forgotten.

Reply:
I don't like changing the same thing twice, especially knowing we will. Can we just change it once and well?

Reply:
No worries, PR updated and force pushed.